ws_api/wealthsimple_api.py

```diff
@@ -214,7 +214,7 @@ def login_internal(self, username: str, password: str, otp_answer: str = None,
         return self.session
 
     def do_graphql_query(self, query_name: str, variables: dict, data_response_path: str, expect_type: str,
-                         filter_fn: callable = None, load_all_pages: bool = False):
+                         filter_fn: callable = None, *, load_all_pages: bool = False):
```

Comment:
Boolean arguments should be keyword-only arguments (forces to specify 'load_all_pages' when using for readability)